bullet_train/docs/stylesheets.md

```diff
@@ -0,0 +1,18 @@
+# Style Sheets
+Bullet Train's stock UI theme, called “Light”, is built to use `tailwindcss` extensively.
```

Comment:
Should we link to https://github.com/bullet-train-co/bullet_train-core/tree/main/bullet_train-themes-light and https://github.com/bullet-train-co/bullet_train-core/tree/main/bullet_train-themes-tailwind_css and give a note about how the Light theme is built on the tailwindcss theme?

Reply:
I've clarified the opening paragraph a bit, but I wouldn't point to the gems here, or away to the Themes doc until the question "but how do I modify these files" pops up as they read the Modify Theme Partials section just below, which points to the Theme doc.

https://github.com/bullet-train-co/bullet_train-core/blob/c8ece1807012351f99980f785f12729f67d772a0/bullet_train/docs/stylesheets.md?plain=1#L1-L2

Reply:
Perfect!